src/services/EdFi.DataManagementService.Api/Core/ApiSchema/ApiSchema_Schema.json

```diff
@@ -1,42 +1,56 @@
 {
-  "type": "object",
-  "properties": {
-    "projectNameMapping": {
-      "$ref": "#/definitions/ProjectNameMapping"
-    },
-    "projectSchemas": {
-      "$ref": "#/definitions/ProjectSchemas"
-    }
-  },
-  "required": [
-    "projectNameMapping",
-    "projectSchemas"
-  ],
-    "definitions": {
-        "ProjectNameMapping": {
+    "type": "object",
+    "properties": {
+        "projectNameMapping": {
             "type": "object",
-            "properties": {
-                "Ed-Fi": {
+            "patternProperties": {
+                "^.*$": {
                     "type": "string"
                 }
-            }
+            },
+            "additionalProperties": false
         },
-        "ProjectSchemas": {
+        "projectSchemas": {
             "type": "object",
-            "properties": {
-                "ed-fi": {
-                    "$ref": "#/definitions/EdFi"
+            "patternProperties": {
+                "^.*$": {
+                    "$ref": "#/definitions/ProjectSchemas"
                 }
-            }
-        },
-        "EdFi": {
+            },
+            "additionalProperties": false
+        }
+    },
+    "required": [
+        "projectNameMapping",
+        "projectSchemas"
+    ],
+    "definitions": {
+        "ProjectSchemas": {
             "type": "object",
             "properties": {
                 "abstractResources": {
-                    "$ref": "#/definitions/AbstractResources"
+                    "type": "object",
```

Comment:
inlined AbstractResources, not limited to EducationOrganization and GeneralStudentProgramAssociation